voxmeld/src/cmd/sound2wem-go/config.json

```diff
@@ -0,0 +1,6 @@
+{
+    "wwisePath": "",
+    "ffmpegPath": "",
+    "projectName": "wavtowemscript",
+    "conversion": "Vorbis Quality High"
```

Comment:
Derzeit wird nur Vorbis als Codec unterstützt - der Codec muss auch in den .bnk Dateien gesetzt werden (das können wir später noch verbessern). Aber die Qualitätsstufe von Vorbis kann man natürlich hier konfigurieren.